inference-engine/src/gna_plugin/transformations/remove_extra_reshapes.hpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace GNAPluginNS {
+
+/**
+ * @brief Removes reshapes before MaxPool which do nothing. Such reshapes can be a result of conversion from IR10 to IR7.
```

Comment:
Did you find the origin of such extra reshapes?

Reply:
They are originated by removal of these patterns (reshape seems to be a result of the merge of the reshapes before and after this pattern):
![image](https://user-images.githubusercontent.com/58477289/116207845-f849c480-a748-11eb-8f12-9d945fcb1dd2.png)